example/spec/parallel/ingest/ingestFromPdrWithUniqueGranuleIdsSpec.js

```diff
@@ -0,0 +1,757 @@
+'use strict';
```

Comment:
This is a boilerplate copy of the existing ingsetFromPdr spec, but updated to show that that test passes if targeting the modified collections and workflows in this PR. 

There's a fair bit of prototype work for CUMULUS-4089 in here 😅, but this is probably needed to make sure that other work in flight doesn't break this PR prior to implementing 4089.

Reply:
Do we need to keep the verification steps which are not related to uniqueGranuleId?

Reply:
@jennyhliu  The question implies the answer, but I think that a large number of the assertions are indirectly tied.   We're testing integration with the changed structure and want that test to validate the full workflow continues working as intended.    I considered looping the original test file, but felt that might be restrictive to the work that was going to be done to make sure the test plan was implemented.       I'll cede the usefulness of that to @npauzenga 's work, but I think we should keep it in place until the test plan work is done.